bi_sql_editor/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2017 - Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'BI SQL Editor',
+    'summary': "BI Views builder, based on Materialized or Normal SQL Views",
```

Comment:
Use single quotes (instead of double quotes) to be consistent with all the rest.